codebuild_cli.py

```diff
@@ -212,29 +224,33 @@ def codedeploy(update, name, desc, git, branch, buildspec, builderimage, release
         'type': 'NO_ARTIFACTS'
     }
 
-    logsConfig={
+    logsConfig = {
         'cloudWatchLogs': {
             'status': 'ENABLED',
         },
     }
-    #breakpoint()
+
     if release:
         pattern = '^refs/tags/.*'
     else:
         pattern = f'^refs/heads/{branch}'
 
     # Either update project or create a new project
     if update:
-        response = client.update_project(
-            name=name,
-            description=desc,
-            source=source,
-            sourceVersion=branch,
-            artifacts=artifacts,
-            environment=environment,
-            serviceRole=role_arn,
-            logsConfig=logsConfig,
-        )
+        try:
+            response = client.update_project(
+                name=name,
+                description=desc,
+                source=source,
+                sourceVersion=branch,
+                artifacts=artifacts,
+                environment=environment,
+                serviceRole=role_arn,
+                logsConfig=logsConfig,
+            )
+        except client.exceptions.ResourceNotFoundException:
+            print("Unable to update a project that does not exist")
+            exit()
```

Comment:
I don't think we should use `exit()` here as we create the project in the `else` block.

Reply:
I put the exit in, as if they want to add a new project and not update, then they should not att the --update flag. I will update the print statement

Reply:
That makes sense!